dumper/app/build.gradle

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2022 Google LLC
+ * Copyright 2023 Google LLC
```

Comment:
Should it be `2023` or `2022-2023`?

Reply:
Let's wait for the CW team's answer.

cc: @shevek-google @paolomorandini